packages/ds/src/admin/layout/Container.tsx

```diff
@@ -37,7 +37,9 @@ export default function Container({
           )}
 
           {/* Actions */}
-          {actions && <div>{actions}</div>}
+          {actions && (
+            <div className="flex items-center justify-end ml-6">{actions}</div>
+          )}
```

Comment:
Tu as raison, on a pas la main si on wrap pour des `<div />` par contre je te propose plus ça comme on a fait ailleurs pour gérer les classes au composant enfant.

```suggestion
          {actions}
```